suppressions.xml

```diff
@@ -28,4 +28,5 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 <suppressions>
     <suppress files="RuntimeTypeAdapterFactory\.java" checks="[a-zA-Z0-9]*"/>
+    <suppress files="Instructions\.java|Obstacles\.java|ClueScroll\.java|ItemStatChanges\.java|JewelleryCharges\.java|Library\.java|PohIcons\.java" checks="AvoidStarImport"/>
```

Comment:
Instead of adding exceptions, I would change the files to either expand the import, or not use static imports and refer the constants through the class name before them